### PR TITLE
Display Bank Name List

### PR DESCRIPTION
**Pull request by MuthuKumari-P**
- Bank name list are displayed using json with spring

### CODE REVIEW

File: src/main/java/com/example/dao/BankDao.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package com.example.dao;
+
+import java.util.List;
+
+public interface BankDao {
```
Comment (selva08sankari) [inline src/main/java/com/example/dao/BankDao.java:5]: add getinstance factory class

File: src/test/java/com/example/Demo2ApplicationTests.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package com.example;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class Demo2ApplicationTests {
```
Comment (selva08sankari) [inline src/test/java/com/example/Demo2ApplicationTests.java:13]: write test case in junit 5

**Review by selva08sankari — APPROVED** [review]

File: src/main/java/com/example/dao/BankDao.java (hunk 1 above)
Comment (MuthuKumari-P) [reply src/main/java/com/example/dao/BankDao.java:5]: I have change it

File: src/test/java/com/example/Demo2ApplicationTests.java (hunk 2 above)
Comment (MuthuKumari-P) [reply src/test/java/com/example/Demo2ApplicationTests.java:13]: I have written it

**Pull request merged**